message_app/__init__.py

```diff
@@ -8,7 +8,7 @@ def create_app(test_config=None):
     app = Flask(__name__)
     app.config.from_mapping(
         SECRET_KEY='dev',
-        DATABASE=os.path.join(app.instance_path, "db", 'message_app_db.sqlite3'),
+        DATABASE=os.path.join("message_app/db", 'message_app_db.sqlite3'),
```

Comment:
Why is this changed to hard coded path `"message_app/db"`?

Reply:
app.instance_path is a path to instance folder in the project directory, and we do not have that folder. In the tutorial, they create database in instance folder.
https://uniwebsidad.com/libros/explore-flask/chapter-5/instance-folder